### PR TITLE
Add a .gitattributes file

### PR DESCRIPTION
**Pull request by spuder**
Add a .gitattributes file to gently coerce windows users to play nice with linux users. 

This shouldn't be a problem if windows dev have set their git config to use core.autocrlf, however in the case when a dev _hasn't_ set that git variable, this will define how to handle line endings correctly. 

Additional information

https://stackoverflow.com/questions/21472971/what-is-the-purpose-of-text-auto-in-gitattributes-file

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
```
Comment (chrivers) [inline .gitattributes:2]: Last line doesn't end in newline :)

Comment (chrivers) [reply .gitattributes:2]: I just looked at the repo, and we only use `*.(py|tpl|stf|txt|md)` - if we set those to `text`, that should do it?

Comment (chrivers) [reply .gitattributes:2]: git might actually understand that syntax, but we should probably make 5 lines, each with a simple glob :)

Comment (spuder) [reply .gitattributes:10]: Ok added specific lines

**Review by chrivers — APPROVED** [review]

**Comment by chrivers** [comment]
Merged, thank you!

**Pull request merged**